watertap/unit_models/reverse_osmosis_base.py

```diff
@@ -370,6 +370,64 @@ def eq_flux_mass(b, t, x, p, j):
                     - prop_perm.conc_mass_phase_comp[p, j]
                 )
 
+        if self.config.transport_model == "SKK":
+
+            self.eq_flux_mass.deactivate()
```

Comment:
I was thinking this implementation would be slightly different in that we would just use the existing `eq_flux_mass` constraint and add refl_coeff bits where needed. Then, you'd have a conditional to check whether SKK or sol-diff, and if sol-diff, fix refl_coeff to 1. To summarize, I was thinking we could use the one flux constraint for both transport models since refl_coeff = 1 in solution-diffusion (so long as it doesn't cause instability and numerous tests associated with RO to fail).

Reply:
Addressed in commit d914a77